### ISSUE TITLE
Make --format group,repo the default

### ISSUE THREAD

**Issue by wmertens** [raineorshine/npm-check-updates#1379]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14.14`

IMHO it's much more user friendly to show repo links and group by update type by default.

**Comment by raineorshine** [comment]
Thank you for the suggestion. I've thought about making `--format group` the default as I tend to prefer it in all cases.

`--format repo` reads all the dependencies' package files from disk which has some performance overhead, so it needs to remain opt-in.

**Comment by CreativeTechGuy** [comment]
> reads all the dependencies' package files from disk which has some performance overhead

It's not doing a directory traversal, it's just loading specific package.json files at known paths when needed. Worst case it'll check 3 paths for every package which got updated and load at most one JSON file.

Given that this package does a lot of fetching over the internet, reading some files off of disk seems pretty fast in comparison I'm not sure it's noticeable.

-----

TLDR: I think it's a good default. :)

**Comment by raineorshine** [comment]
Yeah, you're probably right. And zero overhead if we can parallelize it.

I'm still leaning towards opt-in `--format repo` for aesthetic reasons. (I wish we could do `<a href="">...</a>` in the terminal.) 

`--format group` is definitely better than the current default though.

**Comment by XhmikosR** [comment]
Should `--format group` be the default for interactive mode too?

EDIT: Also, maybe we should show the columns in the order that were specified minus the time?

**Comment by raineorshine** [comment]
> Should `--format group` be the default for interactive mode too?

Yes. We will need to add an option to ungroup though.

> Should `--format group` be the default for interactive mode too?
> 
> EDIT: Also, maybe we should show the columns in the order that were specified minus the time?

I would prefer not to add the complexity unless there is measurable demand for this level of customizability. Good default column order will work for the vast majority of users.
